packages/vm/src/mock/storage.rs

```diff
@@ -0,0 +1,273 @@
+use std::collections::BTreeMap;
+#[cfg(feature = "iterator")]
+use std::{
+    iter,
+    ops::{Bound, RangeBounds},
+};
+
+use crate::{FfiResult, ReadonlyStorage, Storage};
+#[cfg(feature = "iterator")]
+use cosmwasm_std::{Order, KV};
+
+#[derive(Default)]
+pub struct MemoryStorage {
```

Comment:
This is also needed by the crate `cosmwasm-storage`, so I think at least this file needs to remain where it was.

Reply:
Nevermind, it is a different storage now. Makes sense, this is the right direction

Reply:
:+1: